src/features/session_replay/aggregate/index.js

```diff
@@ -221,7 +221,7 @@ export class Aggregate extends AggregateBase {
     }
   }
 
-  prepareHarvest () {
+  prepareHarvest ({ opts } = {}) {
```

Comment:
stylistic tweak, but in accordance with the other 3 features that pass opt into their `prepareHarvest` fn, can this be changed to
```suggestion
  prepareHarvest (options) {
```
and have the schedule pass in `({ ...opts, retry})`

Reply:
Or better yet, just straight pass `{ unload: true}` into the runHarvest call. There's two separate opts-passing system between feature and the runHarvest (which includes the `scheduleHarvest` used on retries) cycle, and this may overwrite something?

Reply:
it isnt always going to be unload: true

Reply:
> There's two separate opts-passing system between feature and the runHarvest (which includes the scheduleHarvest used on retries) cycle, and this may overwrite something?

The opts being held are the same opts that were originally passed into runHarvest, which just means the opts will be preserved for other triggered runHarvests within the same loop